_commands/tools/time.md

```diff
@@ -3,22 +3,55 @@
 
 time
 ---
-TODO: Add documentation for this command by submitting a pull request.
+
 <!-- one line explanation would go here -->
 
+
+`time` measures the time it takes to run a program. It takes at least a command as an argument and outputs the result to either a file or the screen.
+
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ time cd
+
+real 	0m0.008s
+user 	0m0.008s
+sys 	0m0.000s
 ~~~
 
 <!--more-->
 
+**real:** refers to the total time taken to run a command from call to return.
+
+**user:** refers to the amount of time spent in the process called.
+
+**sys:** refers to the amount of time spent in the kernel during the call.
+
 ### Useful Options / Examples
 
 #### Example command
 
-##### Break it down
+##### `time -o [FILE] [COMMAND]`
+`time -o` can be used to put the output of time into a file. This method includes extra information that is not part of the default time. It has cpu usage, inputs and outputs statisics as well. Another thing to note is that the -o command can only be used when using the absolute path of the time command on some installs. Be careful as this command overwrites the contents of FILE.
```

Comment:
ooooh, so `/usr/bin/time` is different than `time`. `time` is a shell builtin command. `/usr/bin/time` is a program like `ls`, `rm`, `grep`, and many others.

I think adding `/usr/bin/time` examples is fine here. But there should probably be a blurb about the difference between `/usr/bin/time` and `time.`